src/components/TableToolbarSelect.js

```diff
@@ -12,26 +12,18 @@ const defaultToolbarSelectStyles = theme => ({
     backgroundColor: theme.palette.background.default,
     flex: '1 1 100%',
     display: 'flex',
-    height: '64px',
     position: 'relative',
     zIndex: 120,
     justifyContent: 'space-between',
+    alignItems: 'center',
     paddingTop: theme.spacing.unit,
     paddingBottom: theme.spacing.unit,
   },
   title: {
     paddingLeft: '26px',
-    top: '50%',
```

Comment:
I think we might need these styles under `title`. If you take a look at this with the `customize-toolbarselect` example, where there are multiple icons in a custom toolbar, you'll see that they don't line up without them.

Reply:
Thanks. Will check it out and update.

Reply:
@gabrielliwerant I don't think we have an issue here. The `customize-toolbarselect` example has the same `iconButton` styling as the default component which is the styling I removed. Removing it from the example and the icons all line up with the title.

Reply:
![image](https://user-images.githubusercontent.com/45109772/59356826-47221880-8cef-11e9-9243-6e8e19bf93a5.png)

Reply:
Lets remove that styling from the example then, so that it is not broken.

Reply:
Example updated

Reply:
Thank you!